packages/sprotty/src/features/edit/edit-label-ui.ts

```diff
@@ -269,7 +269,7 @@ function getEditableLabels(contextElementIds: string[], root: Readonly<SModelRoo
 }
 
 function scaledFont(font: string, zoom: number): string {
-    return font.replace(/([0-9]+)/, (match) => {
+    return font.replace(/([0-9]+[.]?[0-9]*)/, (match) => {
```

Comment:
```suggestion
    return font.replace(/\d+(\.\d+)?/, (match) => {
```

How about this would that work as well? I think your expression would also match '1.'.

Reply:
Good point! I'll use this one instead.